cardano-db-sync/src/Cardano/DbSync/Validator.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
`RecordWildCards` is potentially bad for readability and is not use anywhere in `db-sync` . It should be removed.

This code in this file also seems to belong in the `cardano-db-tool`  app and be in the `cardano-db/app/Cardano/Db/App` part of the tree.